python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -468,16 +451,20 @@ def __set__(self, obj, val):
         if hasattr(obj, '_checkset_name'):
             checkSet = self._bind_method(obj, self._checkset_name)
             if checkSet is not None:
-                checkSet(_val)
+                checkSet(val)
+
         if hasattr(obj, '_filter_name'):
             this_filter = self._bind_method(obj, self._filter_name)
             if this_filter:
-                val = this_filter(_val)
+                val = this_filter(val)
```

Comment:
This happened recently with the removal of `__atomic_set__` but these features are hardly ever used and in fact, this code doesn't work anyway as it's asking for `_filter_name` and `_checkset_name` on `obj` rather than `self`. These can basically be completely removed but I thought it better to wait until we close #616